src/main/resources/application.properties

```
@@ -1 +1,12 @@
-app.name=Sistema de Gestión de Biblioteca
\ No newline at end of file
+app.name=Sistema de Gestión de Biblioteca
+# H2 Database
+spring.datasource.url=jdbc:h2:mem:biblioteca
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=
+spring.h2.console.enabled=true
+spring.h2.console.path=/h2-console
+spring.sql.init.mode=always
+# JPA/Hibernate
+spring.jpa.show-sql=true
+spring.jpa.hibernate.ddl-auto=update
```

Muchas de esas propiedades sobran

```suggestion
spring.datasource.url=jdbc:h2:mem:biblioteca
spring.datasource.driverClassName=org.h2.Driver
spring.datasource.username=sa
spring.datasource.password=
```